installing/installing_gcp/installing-gcp-user-infra.adoc

```diff
@@ -16,8 +16,9 @@ The steps for performing a user-provisioned infrastructure installation are prov
 
 == Prerequisites
 
-* Review details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
-* If you use a firewall and plan to use telemetry, you must xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure the firewall to allow the sites] that your cluster requires access to.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* If you use a firewall and plan to use telemetry, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] that your cluster requires access to.
```

Comment:
No prereq to configure GCP account here?

Reply:
I have requested a review of this outside of this PR, as detailed in my comment above.